spec/unit/serialize/v2/http/builds_spec.rb

```diff
@@ -80,7 +80,6 @@
   end
 
   it 'queries' do
-    lambda { data }.should issue_queries(9)
+    lambda { data }.should issue_queries(5)
```

Comment:
Not familiar with this spec – is this a case of newer activerecord doing the same thing in fewer queries?

Reply:
yep :)